### ISSUE TITLE
Pull faculty names and NET IDs from ESD

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/laevigata#435]
- [x] Data from ESD is being exported to a JSON file 
- [ ] Emory Committee Member field should populated from this file
- [ ] Emory Committee Chair/Thesis Advisor field should be populated with this file

**Comment by shanitajohn** [comment]
Work was completed as a part of #483

**Comment by val99erie** [comment]
Blocked by work on committee member and committee chair fields in story #474.

**Comment by val99erie** [comment]
This story will be unblocked after this PR is merged into master:  https://github.com/curationexperts/laevigata/pull/621

**Comment by shanitajohn** [comment]
We should know how this work needs to be implemented, but this functionality is not required for launch.

**Comment by bess** [comment]
@shanitajohn and @rotated8  I understood we were putting this on hold for awhile while Emory creates a local authority for these. Is that right? If so, can we move this back to the backlog?

**Comment by shanitajohn** [comment]
@bess Yes, that was my understanding, but I would love confirmation from @rotated8, too.

**Comment by rotated8** [comment]
Yep, sounds good!

**Comment by mark-dce** [comment]
ESD was never implemented.
